client-side/src/components/profileComponents/ProfileActions.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Tooltip, Button } from '@mui/material';
+import { TOOLTIP_TEXTS, BUTTON_LABELS } from '../../constants/profileConstants.js';
+
+export default function ProfileActions({ handleSave, handleClose, handleDelete }) {
```

Comment:
1. you added prop types but you didn't use them as well, you exported the component before defined the prop types.
2. please use defaultParameters instead of proptypes.defaultValues - this is depreceted

do t for all relevant components